web-common/src/components/navigation/breadcrumbs/BreadcrumbItem.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+  import * as DropdownMenu from "@rilldata/web-common/components/dropdown-menu";
+  import CaretDownIcon from "@rilldata/web-common/components/icons/CaretDownIcon.svelte";
+  import Check from "@rilldata/web-common/components/icons/Check.svelte";
+  import { page } from "$app/stores";
+  import type { Level } from "./Breadcrumbs.svelte";
+
+  const regex = /\[.*?\]/g;
+
+  export let options: Level;
+  export let current: string;
+  export let isCurrentPage = false;
+  export let depth: number = 0;
+  export let onSelectMenuOption: undefined | ((id: string) => void) = undefined;
+
+  $: selected = options.get(current);
+  $: nextChild = $page.route.id?.split(regex)[depth + 1];
+</script>
+
+<li class="flex items-center gap-x-2 px-2">
+  <div class="flex flex-row gap-x-1 items-center">
+    {#if selected}
+      <a
+        href={!isCurrentPage && selected.href
+          ? selected.href + nextChild
+          : "#top"}
```

Comment:
Confirming, it seems like "#top" is supposed to give the user feedback that they did indeed click the link, even though the link doesn't direct them anywhere?

Reply:
I think it probably doesn't really matter, but #top is just acknowledging that the element is a link that references the current page as opposed to a fully qualified link that happens to be the same as the page you're on.